<?xml version="1.0" standalone="yes"?>
<NewDataSet>
  <xs:schema id="NewDataSet" xmlns="" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
    <xs:element name="NewDataSet" msdata:IsDataSet="true" msdata:MainDataTable="Table1" msdata:UseCurrentLocale="true">
      <xs:complexType>
        <xs:choice minOccurs="0" maxOccurs="unbounded">
          <xs:element name="Table1">
            <xs:complexType>
              <xs:sequence>
                <xs:element name="Error_Type" type="xs:string" minOccurs="0" />
                <xs:element name="Error_DateTime" type="xs:dateTime" minOccurs="0" />
                <xs:element name="Error_Description" type="xs:string" minOccurs="0" />
                <xs:element name="Error_Source" type="xs:string" minOccurs="0" />
                <xs:element name="Error_Stack_Trace" type="xs:string" minOccurs="0" />
                <xs:element name="Error_Caused_By_Method" type="xs:string" minOccurs="0" />
                <xs:element name="Optional_Text_1" type="xs:string" minOccurs="0" />
                <xs:element name="Optional_Text_2" type="xs:string" minOccurs="0" />
                <xs:element name="Optional_Text_3" type="xs:string" minOccurs="0" />
                <xs:element name="NombreUsuario" type="xs:string" minOccurs="0" />
                <xs:element name="HostName" type="xs:string" minOccurs="0" />
                <xs:element name="IPAddress" type="xs:string" minOccurs="0" />
              </xs:sequence>
            </xs:complexType>
          </xs:element>
        </xs:choice>
      </xs:complexType>
    </xs:element>
  </xs:schema>
  <Table1>
    <Error_Type>System.Collections.ListDictionaryInternal</Error_Type>
    <Error_DateTime>2021-02-23T14:41:00.4055336-05:00</Error_DateTime>
    <Error_Description>Índice fuera de los límites de la matriz.</Error_Description>
    <Error_Source>AplicacionSistemaVentura</Error_Source>
    <Error_Stack_Trace>   en AplicacionSistemaVentura.InterfazPrincipal.VerificarAlertas() en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\AplicacionSistemaVentura\InterfazPrincipal.xaml.cs:línea 257</Error_Stack_Trace>
    <Error_Caused_By_Method>Void VerificarAlertas()</Error_Caused_By_Method>
    <Optional_Text_1 />
    <Optional_Text_2 />
    <Optional_Text_3 />
    <NombreUsuario>MANAGER</NombreUsuario>
    <HostName>WIN-Q3O0BSPVASQ</HostName>
    <IPAddress>192.168.1.40</IPAddress>
  </Table1>
  <Table1>
    <Error_Type>System.Collections.ListDictionaryInternal</Error_Type>
    <Error_DateTime>2021-02-23T15:19:41.7825764-05:00</Error_DateTime>
    <Error_Description>La columna 'IdTLlantas' no pertenece a la tabla .</Error_Description>
    <Error_Source>System.Data</Error_Source>
    <Error_Stack_Trace>   en System.Data.DataRow.GetDataColumn(String columnName)
   en System.Data.DataRow.get_Item(String columnName)
   en AplicacionSistemaVentura.PAQ01_Definicion.GestNeumatico.LlenarDatos(Int32 IdNeumatico) en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\AplicacionSistemaVentura\PAQ01-Definicion\GestNeumatico.xaml.cs:línea 581</Error_Stack_Trace>
    <Error_Caused_By_Method>System.Data.DataColumn GetDataColumn(System.String)</Error_Caused_By_Method>
    <Optional_Text_1 />
    <Optional_Text_2 />
    <Optional_Text_3 />
    <NombreUsuario>MANAGER</NombreUsuario>
    <HostName>WIN-Q3O0BSPVASQ</HostName>
    <IPAddress>192.168.1.40</IPAddress>
  </Table1>
  <Table1>
    <Error_Type>System.Collections.ListDictionaryInternal</Error_Type>
    <Error_DateTime>2021-02-23T15:40:22.0829927-05:00</Error_DateTime>
    <Error_Description>Índice fuera de los límites de la matriz.</Error_Description>
    <Error_Source>AplicacionSistemaVentura</Error_Source>
    <Error_Stack_Trace>   en AplicacionSistemaVentura.InterfazPrincipal.VerificarAlertas() en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\AplicacionSistemaVentura\InterfazPrincipal.xaml.cs:línea 257</Error_Stack_Trace>
    <Error_Caused_By_Method>Void VerificarAlertas()</Error_Caused_By_Method>
    <Optional_Text_1 />
    <Optional_Text_2 />
    <Optional_Text_3 />
    <NombreUsuario>MANAGER</NombreUsuario>
    <HostName>WIN-Q3O0BSPVASQ</HostName>
    <IPAddress>192.168.1.40</IPAddress>
  </Table1>
  <Table1>
    <Error_Type>System.Collections.ListDictionaryInternal</Error_Type>
    <Error_DateTime>2021-02-23T16:19:16.5085172-05:00</Error_DateTime>
    <Error_Description>Procedure or function 'Neumatico_UpdateCascade' expects parameter '@TLlanta', which was not supplied.</Error_Description>
    <Error_Source>.Net SqlClient Data Provider</Error_Source>
    <Error_Stack_Trace>   en System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   en System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   en System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   en System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean&amp; dataReady)
   en System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   en System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task&amp; task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task&amp; task, Boolean&amp; usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, String methodName, Boolean sendToPipe, Int32 timeout, Boolean&amp; usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.ExecuteNonQuery()
   en Data.D_Neumatico.Neumatico_UpdateCascade(E_Neumatico E_Neumatico, DataTable tblCiclo) en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\Data\D_Neumatico.cs:línea 183
   en Business.B_Neumatico.Neumatico_UpdateCascade(E_Neumatico E_Neumatico, DataTable tblCiclo) en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\Business\B_Neumatico.cs:línea 49
   en AplicacionSistemaVentura.PAQ01_Definicion.GestNeumatico.BtnGrabar_Click(Object sender, RoutedEventArgs e) en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\AplicacionSistemaVentura\PAQ01-Definicion\GestNeumatico.xaml.cs:línea 251</Error_Stack_Trace>
    <Error_Caused_By_Method>Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])</Error_Caused_By_Method>
    <Optional_Text_1 />
    <Optional_Text_2 />
    <Optional_Text_3 />
    <NombreUsuario>MANAGER</NombreUsuario>
    <HostName>WIN-Q3O0BSPVASQ</HostName>
    <IPAddress>192.168.1.40</IPAddress>
  </Table1>
  <Table1>
    <Error_Type>System.Collections.ListDictionaryInternal</Error_Type>
    <Error_DateTime>2021-02-23T17:07:42.4125931-05:00</Error_DateTime>
    <Error_Description>The INSERT statement conflicted with the FOREIGN KEY constraint "FK_ProgramacionDetalle_Programacion". The conflict occurred in database "VSMONE", table "dbo.Programacion", column 'IdProgramacion'.
-----------------------------
Insertar Programacion
-----------------------------
Actualizar Bitacora
-----------------------------
Insertar Programacion Detalle
The statement has been terminated.
IdError
547</Error_Description>
    <Error_Source>.Net SqlClient Data Provider</Error_Source>
    <Error_Stack_Trace>   en System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   en System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   en System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   en System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean&amp; dataReady)
   en System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   en System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task&amp; task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task&amp; task, Boolean&amp; usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, String methodName, Boolean sendToPipe, Int32 timeout, Boolean&amp; usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.ExecuteNonQuery()
   en Data.D_Programacion.Programacion_UpdateCascade(E_Programacion E_Programacion, DataTable tblBitacora, DataTable tblProgramacionDet) en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\Data\D_Programacion.cs:línea 128
   en Business.B_Programacion.Programacion_UpdateCascade(E_Programacion E_Programacion, DataTable tblBitacora, DataTable tblProgramacionDet) en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\Business\B_Programacion.cs:línea 34
   en AplicacionSistemaVentura.PAQ02_Planificacion.PlanProgramacionUC.worker_RunWorkerCompleted(Object sender, RunWorkerCompletedEventArgs e) en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\AplicacionSistemaVentura\PAQ02-Planificacion\PlanProgramacionUC.xaml.cs:línea 1491</Error_Stack_Trace>
    <Error_Caused_By_Method>Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])</Error_Caused_By_Method>
    <Optional_Text_1 />
    <Optional_Text_2 />
    <Optional_Text_3 />
    <NombreUsuario>MANAGER</NombreUsuario>
    <HostName>WIN-Q3O0BSPVASQ</HostName>
    <IPAddress>192.168.1.40</IPAddress>
  </Table1>
  <Table1>
    <Error_Type>System.Collections.ListDictionaryInternal</Error_Type>
    <Error_DateTime>2021-02-23T17:36:37.1089289-05:00</Error_DateTime>
    <Error_Description>Índice fuera de los límites de la matriz.</Error_Description>
    <Error_Source>AplicacionSistemaVentura</Error_Source>
    <Error_Stack_Trace>   en AplicacionSistemaVentura.InterfazPrincipal.VerificarAlertas() en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\AplicacionSistemaVentura\InterfazPrincipal.xaml.cs:línea 257</Error_Stack_Trace>
    <Error_Caused_By_Method>Void VerificarAlertas()</Error_Caused_By_Method>
    <Optional_Text_1 />
    <Optional_Text_2 />
    <Optional_Text_3 />
    <NombreUsuario>MANAGER</NombreUsuario>
    <HostName>WIN-Q3O0BSPVASQ</HostName>
    <IPAddress>192.168.1.40</IPAddress>
  </Table1>
</NewDataSet>